code/modules/mob/living/carbon/carbon.dm

```diff
@@ -3,7 +3,7 @@
 	update_hud()
 	return
 
-/mob/living/carbon/Bump(var/atom/movable/AM, yes)
+/mob/living/carbon/Bump(var/atom/movable/AM, yes = 1)
```

Comment:
I assume `yes` defaulted to null previously because we wanted to ignore the duplicate `Bump()` calls, considering that literally every `Bump()` call in the code except one calls it with 1, so it can be safely set to default to 1 now that all the duplicate calls are gone.

If I'm wrong and this was important, please yell at me.